spec/active_record_consume/mass_updater_spec.rb

```diff
@@ -39,6 +39,8 @@
       end
     end
 
+    let(:bulk_id_generator) { proc { SecureRandom.uuid } }
```

Comment:
Is this actually tested in the spec? This is identical to the default so how can we validate that it's actually being called?